config/cloud.cfg.tmpl

```diff
@@ -156,6 +165,9 @@ cloud_config_modules:
 {% endif %}
   - locale
 {% endif %}
+{% if variant == "raspberry-pi-os" %}
```

Comment:
`rpi_interfaces` was also defined in the network stage. Does this module need to run twice? Why?

Reply:
You're right, it doesn't. I'll remove the duplicate.